project/lang/en.php

```diff
@@ -5,24 +5,23 @@ class Lang extends DefaultLang
 {
 	var $js = array
 	(
-		'confirmDelete' => "Are you sure you want to delete the task?",
+		'confirmDelete' => "Are you sure you want to delete the story?",
```

Comment:
What's the difference of these two files ?

Reply:
we decide to use "story" rather than "task" in our app